project/PekkoDisciplinePlugin.scala

```diff
@@ -74,16 +74,19 @@ object PekkoDisciplinePlugin extends AutoPlugin {
     "pekko-stream-tests-tck",
     "pekko-testkit")
 
-  val defaultScalaOptions = "-Wconf:cat=unused-nowarn:s,any:e"
+  lazy val defaultScalaOptions = Def.setting(CrossVersion.partialVersion(scalaVersion.value) match {
+    case Some((2, 12)) => "-Wconf:cat=unused-nowarn:s,any:e"
+    case _             => "-Wconf:cat=unused-nowarn:s,cat=lint-named-booleans:s,cat=other-shadowing:s,any:e"
```

Comment:
was it impossible to fix the warnings?

Reply:
Possible, but then need to change a lot of code.

Reply:
some of the code that doesn't compile appears to be generated - I think we may need to skip this upgrade and report bugs to Scala team

Reply:
gotcha - maybe create an issue for it here for starters and refer to that from here?